libsentrykube/quickpatch.py

```diff
@@ -92,6 +91,27 @@ def get_arguments(service: str, patch: str) -> Sequence[str]:
     return patch_data["schema"].get("required", [])
 
 
+def patch_json(patch_data: List[dict], resource_data: dict) -> dict:
```

Comment:
The dict type for resource is because the output type of get_tools_managed_service_value_overrides() is also a dict so I needed to use that to avoid type errors

Reply:
Then let's fix `get_tools_managed_service_value_overrides` . It was a mistake.